evaluations.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Evaluations-MockUp</title>
+		<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+		<link rel="stylesheet" type="text/css" href="css/evaluations.css">
+		<link rel="stylesheet" type="text/css" href="css/common.css">
+	</head>
+	<body>
+	<!-- Header --> 
+		<header class="header">
+			<nav class="nav">
+				<div id="logo" class="logo "></div>
+				<div id="navigation" class="navigationClass">
+					<div id="button1" class="button button1 formatted selected">
+					Evaluations
+					</div>
+					<a href="new.html"><div id="button2" class="button button2 formatted">
```

Comment:
Why do you have a `div` in an `a` tag? Is it required? Can you describe what you're trying to do?